### PR TITLE
Sign Up page and eye button in password field

### PR DESCRIPTION
**Pull request by macck7**

### CODE REVIEW

**Comment by macck7** [comment]
Sign Up page and eye button in password field funtionality

**Review by cehicm — CHANGES_REQUESTED** [review]
Hi @macck7, please don't leave comments behind, this can be uncommented. Thanks!

File: components/Alert.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+function Alert(props) {
+  
+    const capital=(text)=>{
+        var tt  = text.toLowerCase();
+        return tt.charAt(0).toUpperCase()+tt.slice(1);
+    }
+
+  return (
+      <div style={{ height : '50px' }}>
```
Comment (cehicm) [inline components/Alert.js:11]: We're using Tailwind on the front end, rather than inline styles. Please update with classes here.

**Comment by nikolap994** [comment]
To make everything cleaner, make new pull request with @cehicm  changes.

File: components/Alert.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function Alert(props) {
+  
+    const capital=(text)=>{
+        var tt  = text.toLowerCase();
```
Comment (cehicm) [inline components/Alert.js:6]: 1. `var` isn't ideal here (or in general). Please change. For more info, check out [this article.](https://blog.devgenius.io/let-var-and-const-in-js-2f6226457d2f)
2.  `tt` as a name isn't very understandable, please change.

**Pull request merged**